src/content-scripts/ActionMgr.js

```diff
@@ -12,22 +12,18 @@ class ActionMgr {
 	 * @param {string[]} links
 	 */
 	CopyToClipboard(links) {
-		if(Prefs.DevMode && Prefs.Dev_Skip_AllActions)
-			return console.log('Skipped Copying Links: %o', links);
-
-		const input = document.createElement('textarea');
+	    if (Prefs.DevMode && Prefs.Dev_Skip_AllActions)
+	        return console.log('Skipped Copying Links: %o', links);
 
-		input.style.position = 'fixed';
-		input.style.opacity  = '0';
-		input.value          = links.join('\n');
+	    let joinedLinks = links.join('\n');
 
-		document.body.appendChild(input);
-		input.select();
-
-		document.execCommand('Copy');
-		document.body.removeChild(input);
+	    navigator.clipboard.writeText(joinedLinks)
+	    .then(function () {
+	        /* clipboard successfully set */
+	    }, function () {
+	        /* clipboard write failed */
+	    });
 	}
-
```

Comment:
I might not have understand well [this page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard). But what I understood is that `navigator.clipboard.writeText()` needs more permissions and works only on https pages. If that's so I don't think we should make this change.

Reply:
I didn't notice that actionmanager.js is a "content script"..
I will reverse the change

Reply:
It might not be the case, my understanding of this is sparse. I'm wondering if that's the case but if you think it's not I'll trust you on that.